lib/Elevate/Blockers/Repositories.pm

```diff
@@ -257,6 +258,23 @@ sub _check_yum_repos ($self) {
                             $current_repo_name, $path
                         )
                     );
+                    my $blocker_id = ref($self) . '::YumInstalledPackagesFromUnsupportedRepository';
```

Comment:
If this is replacing the unsupported repo stuff, then I'd rather we leave the previous alone. This is going to aggregate all the repos into 1 category, and that's definitely undesirable.

Reply:
According to the information I had in the case, this seemed like a good method for aggregating the data on packages installed from an unsupported repository.